apps/ehr/src/features/in-house-labs/components/details/CollectSampleView.tsx

```diff
@@ -70,18 +69,16 @@ export const CollectSampleView: React.FC<CollectSampleViewProps> = ({ testDetail
     setShowSampleCollection(!showSampleCollection);
   };
 
-  const handleToggleDetails = (): void => {
-    setShowDetails(!showDetails);
-  };
-
   const handleMarkAsCollected = (): void => {
     onSubmit({
       specimen: {
         source: sourceType,
         collectedBy: { id: collectedById, name: providers.find((p) => p.id === collectedById)?.name || '' },
         collectionDate: date.toISO(),
       },
-      notes,
+      // todo - will this be editable in the future? if no we should just remove it from the params
+      // right now notes are readonly on the sample collection page
```

Comment:
That makes sense. Ideally, if we remove this field, we should also update the zambda since the field can still be sent from the frontend. However, I think removing this code now would be an improvement over the current situation, as it reduces the risk of accidentally overwriting data due to frontend errors.